x/amm/keeper/keeper_exit_pool.go

```diff
@@ -40,5 +40,8 @@ func (k Keeper) ExitPool(
 		return sdk.Coins{}, err
 	}
 
+	// Decrease liquidty amount
+	k.RecordTotalLiquidityIncrease(ctx, exitCoins)
```

Comment:
@kenta-elys are we suppose to increase or decrease the liquidity in the exit pool function?

Reply:
@cosmic-vagabond 
Yes I did it like that. I could see there is one global denom liquidity across all pools. We should account it properly as there is the place to check with it.